OGGExtractor.cpp

```diff
@@ -259,7 +265,10 @@ void OGGExtractor::scanContainers()
 
   m_thread->start();
 }
-
+QString OGGExtractor::getDefaultOutputFilename(const int i,const OGGData& data)
+{
+  return tr("%1 0x%2-0x%3 (%4)").arg(data.container.split('/').last().split('.').first()).arg(data.start, 0, 16).arg(data.end, 0, 16).arg(data.end - data.start); 
```

Comment:
Maybe will be better to use a QFile.
**QFile container{data.container};**
and then **container.filename().split(".").first()** to get parameter %1 and avoid using the character '/' . It's not the separator in Windows.

Reply:
Or use QDir::fromNativeSeparators to ensure that '/' is the separator . I don't remember the format of the container string, maybe none of this is necessary.